### ISSUE TITLE
CLI option names: Review

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/spoonbill#111]
Some option names could perhaps be improved: https://open-contracting.github.io/spoonbill/cli.html

@sabahfromlondon @yolile What do you think of?

* `--threshold NUM` -> `--split-at NUM`, which communicates what action happens at the threshold
* `--selection TABLE,TABLE` -> `--only-initial-tables TABLE,TABLE`, which clarifies that this is only for initial tables  (described in the docs as the top-level planning, tenders, awards, contracts tables), not child tables or columns
* `--exclude TABLE,TABLE` -> `--exclude-child-tables TABLE,TABLE`, which clarifies that this is only for child tables, not initial tables or columns

I think all other option names are clear.

**Comment by sabahfromlondon** [comment]
@jpmckinney I like the suggested changes a lot. `split-at NUM` for instance is much clearer.

The only suggestion I have is that maybe instead of `initial tables` use the wording `root-tables`?

**Comment by yolile** [comment]
I agree with @sabahfromlondon, and another option instead of initial tables is "parent" tables, which makes sense as we are using "child-tables" later.

**Comment by jpmckinney** [comment]
I'm happy with `--only-root-tables`. `--only-parent-tables` could be confusing, because `contracts_items` is a parent table to `contracts_items_class`, but it is not an initial/root table.

**Comment by jpmckinney** [comment]
@sorenabell This is the sort of small change that I am happy to commit on my own, so that Quinta can focus on the more challenging and time-consuming changes.

For this kind of change, do you prefer that I assign the PR for review by Quinta, or just merge it myself?

**Comment by sorenabell** [comment]
@jpmckinney please assign PR review for @yshalenyk nevertheless, just for us to be in the loop.
